File: app/src/main/java/in/woloo/www/refer_woloo_host/model/ReferWolooResponse.java
package in.woloo.www.refer_woloo_host.model;

import com.google.gson.annotations.SerializedName;

public class ReferWolooResponse{

	@SerializedName("data")
	private Data data;

	public Data getData(){
		return data;
	}

	public class Data{

		@SerializedName("message")
		private String message;

		@SerializedName("status")
		private String status;

		public String getMessage(){
			return message;
		}

		public String getStatus(){
			return status;
		}
	}
}

File: app/src/main/java/in/woloo/www/more/refer_woloo_host/model/ReferWolooResponse.kt
package `in`.woloo.www.more.refer_woloo_host.model

import com.google.gson.annotations.SerializedName

class ReferWolooResponse {
    @SerializedName("data")
    val data: Data? = null

    inner class Data {
        @SerializedName("message")
        val message: String? = null

        @SerializedName("status")
        val status: String? = null
    }
}